client/analytics/report/revenue/index.js

```diff
@@ -212,12 +233,59 @@ class RevenueReport extends Component {
 		];
 	}
 
+	// TODO since this pattern will exist on every report, this possibly should become a component
+	getChartSummaryNumbers() {
+		const { totals = {} } = this.state.stats;
+		const selectedChart = this.getSelectedChart();
+
+		const summaryNumbers = map( charts, ( chart, key ) => {
+			const { label, type } = chart;
+			const isSelected = selectedChart === key;
+			let value = totals[ key ];
+
+			switch ( type ) {
+				// TODO: implement other format handlers
+				case 'currency':
+					value = formatCurrency( value );
+					break;
+			}
+
+			const onClick = () => {
+				this.onQueryChange( 'chart' )( key );
+			};
+
+			return (
+				<SummaryNumber
+					key={ key }
+					value={ value }
+					label={ label }
+					selected={ isSelected }
+					delta={ 0 }
+					onToggle={ onClick }
```

Comment:
`onToggle` wasn't meant to be used like this, the idea was that each SummaryNumber would use the `href` prop with `getNewPath` so it's a link, not a button.

Reply:
doh! i'll update in a follow-up PR.